### PR TITLE
Left menu changes

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description

Changes in the left menu order
Change of styles in Home > Overview
Changed left menu and breadcrumbs names
Redirects were changed so that it does not refresh the page and to have it centralized in the applications file.
 
### Issues Resolved

- https://github.com/wazuh/wazuh-dashboard/issues/94

### Evidence

image
image
image

### Test

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome |
| --- |  --- |
| See the menu titles without camel case and the sub sections in camel case, in the breadcrumb it should look the same.| :black_circle: |
| The Home > Overview icon styles should be the same as those in the menu (test it in dark and light mode) | :black_circle: |
| Breadcrumb redirect well to the agent | :black_circle: |
| In the agent view: The pinned applications should redirect to the respective application without reloading the page and if you hover it should show the url to which it redirects. | :black_circle: |
| In the agent view: When the menu is opened with the `More` button the options should redirect to the respective application without reloading the page and if hovered the url to which it redirects should appear. | :black_circle: |
| In the agent view: The Open MITRE ATT&CK button should redirect to the MITRE ATT&CK application without reloading the page and if hovered the url to which it redirects should appear. | :black_circle: |
| In the agent view: On the `FIM` card the icon should redirect to `FIM` without reloading the page and if hovered the url should appear. | :black_circle: |
| In the agent view: On the `SCA: last scan` card the icon should redirect to `SCA` without reloading the page and if the cursor is hovered over the url it redirects to should appear. | :black_circle: |
| In the agent view: On the `SCA: last scan` card the title should redirect to `SCA` without reloading the page and hovering over it should display the url it redirects to. | :black_circle: |
| In the agent view: On the `SCA: last scan` card The caption should redirect to `SCA` without reloading the page and if hovered over should display the url to which it redirects. | :black_circle: |
| In the overview: Without agents the `Add agent` button should redirect to the `Endpoint Summary` application without making a page reload and if hovered the url to which it redirects should appear. | :black_circle: |
| In the overview:  The cards should redirect to their respective application without reloading the page and if you hover on the title you should see the url you are redirected to. | :black_circle: |
| In the healthcheck: The `Go to settings` button should redirect to the `Server APIs` application without reloading the page and if you hover it should show the url to which it redirects. | :black_circle: |
| Applications with sample data: The button that takes you to sample data from the applications should redirect to the `Sample Data` application without making a page reload and if you hover it should appear the url to which it redirects. | :black_circle: |
| In Endpoint Summary: The action buttons of the table should redirect to the `IT Hygiene` application without reloading the page and if hovered the url to which it redirects should appear. | :black_circle: |
| In the configuration of an agent: The group buttons should redirect to the `Endpoint Groups` application without reloading the page and if hovered the url should be displayed. | :black_circle: |
| In statistics: The `Settings` button should redirect to the `App Settings` application without a page reload and if hovered the url to which it redirects should appear. | :black_circle: |

**Details**
<details>
<summary>:black_circle: See the menu titles without camel case and the sub sections in camel case, in the breadcrumb it should look the same.
</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: The Home > Overview icon styles should be the same as those in the menu (test it in dark and light mode)</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: Breadcrumb redirect well to the agent</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: The pinned applications should redirect to the respective application without reloading the page and if you hover it should show the url to which it redirects.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: When the menu is opened with the `More` button the options should redirect to the respective application without reloading the page and if hovered the url to which it redirects should appear.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: The Open MITRE ATT&CK button should redirect to the MITRE ATT&CK application without reloading the page and if hovered the url to which it redirects should appear.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: On the `FIM` card the icon should redirect to `FIM` without reloading the page and if hovered the url should appear.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: On the `SCA: last scan` card the icon should redirect to `SCA` without reloading the page and if the cursor is hovered over the url it redirects to should appear.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: On the `SCA: last scan` card the title should redirect to `SCA` without reloading the page and hovering over it should display the url it redirects to.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: On the `SCA: last scan` card The caption should redirect to `SCA` without reloading the page and if hovered over should display the url to which it redirects.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the overview: Without agents the `Add agent` button should redirect to the `Endpoint Summary` application without making a page reload and if hovered the url to which it redirects should appear.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the overview:  The cards should redirect to their respective application without reloading the page and if you hover on the title you should see the url you are redirected to.
</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the healthcheck: The `Go to settings` button should redirect to the `Server APIs` application without reloading the page and if you hover it should show the url to which it redirects.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: - Applications with sample data: The button that takes you to sample data from the applications should redirect to the `Sample Data` application without making a page reload and if you hover it should appear the url to which it redirects.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: - In Endpoint Summary: The action buttons of the table should redirect to the `IT Hygiene` application without reloading the page and if hovered the url to which it redirects should appear.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: - In the configuration of an agent: The group buttons should redirect to the `Endpoint Groups` application without reloading the page and if hovered the url should be displayed.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: - In statistics: The `Settings` button should redirect to the `App Settings` application without a page reload and if hovered the url to which it redirects should appear.</summary>

Chrome - :black_circle:

</details>

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: plugins/main/public/components/common/modules/overview-current-section.tsx [hunk 1]
```diff
@@ -60,16 +65,7 @@ class WzCurrentOverviewSection extends Component {
...
-    return (
-      <span>
-        {/*this.props.currentTab && WAZUH_MODULES[this.props.currentTab] && WAZUH_MODULES[this.props.currentTab].title && (
-      <EuiTitle size='s'>
-        <h2>
-          {WAZUH_MODULES[this.props.currentTab].title}
-       </h2>
-      </EuiTitle>)*/}
-      </span>
-    );
```
Comment (Desvelao) [inline plugins/main/public/components/common/modules/overview-current-section.tsx:72]: **thought**: is this being rendered before the change?

File: plugins/main/public/styles/index.ts [hunk 2]
```diff
@@ -27,7 +27,5 @@ const IS_DARK_THEME = getUiSettings().get('theme:darkMode');
 /* tslint-disable no-undef */
 if (IS_DARK_THEME) {
   import('./theme/dark/index.dark.scss').then();
-}else{
-  import('./theme/light/index.light.scss').then();
```
Comment (Desvelao) [inline plugins/main/public/styles/index.ts:31]: **thought**: why is this removed?

File: plugins/main/public/utils/applications.ts [hunk 3]
```diff
@@ -670,7 +670,7 @@ export const serverApi = {
   category: 'management',
   id: 'server-api',
```
Comment (Desvelao) [inline plugins/main/public/utils/applications.ts:671]: **nitpick**: change to `server-apis`.

File: plugins/main/public/utils/applications.ts [hunk 4]
```diff
@@ -569,7 +569,7 @@ const statistics = {
   redirectTo: () => '/manager/?tab=statistics',
 };
 
-const logs = {
+export const logs = {
   category: 'wz-category-server-management',
   id: 'logs',
```
Comment (Desvelao) [inline plugins/main/public/utils/applications.ts:574]: **nitpick**: rename to `app-logs`.

File: plugins/main/public/utils/applications.ts [hunk 5]
```diff
@@ -601,7 +601,7 @@ const reporting = {
   redirectTo: () => '/manager/?tab=reporting',
 };
 
-const settings = {
+export const settings = {
   category: 'wz-category-server-management',
   id: 'settings',
```
Comment (Desvelao) [inline plugins/main/public/utils/applications.ts:606]: **nitpick**: rename to `app-settings`.

File: plugins/main/public/utils/applications.ts [hunk 6]
```diff
@@ -702,7 +702,7 @@ export const wazuhPluginSettings = {
   category: 'management',
   id: 'wazuh-plugin-settings',
```
Comment (Desvelao) [inline plugins/main/public/utils/applications.ts:703]: **nitpick**: rename to `app-settings`.

File: plugins/main/public/utils/applications.ts [hunk 7]
```diff
@@ -718,7 +718,7 @@ const wazuhPluginLogs = {
   category: 'management',
   id: 'wazuh-plugin-logs',
```
Comment (Desvelao) [inline plugins/main/public/utils/applications.ts:719]: **nitpick**: rename to `app-logs`.

File: plugins/main/public/utils/applications.ts [hunk 8]
```diff
@@ -734,7 +734,7 @@ const wazuhPluginAbout = {
   category: 'management',
   id: 'wazuh-plugin-about',
```
Comment (Desvelao) [inline plugins/main/public/utils/applications.ts:735]: **nitpick**: rename to `about`.

File: plugins/main/public/components/common/modules/overview-current-section.tsx (hunk 1 above)
Comment (yenienserrano) [reply plugins/main/public/components/common/modules/overview-current-section.tsx:72]: I understand that this code is commented and that it is not rendered, or am I wrong?

File: plugins/main/public/styles/index.ts (hunk 2 above)
Comment (yenienserrano) [reply plugins/main/public/styles/index.ts:31]: The only thing it contained was the `Home -> Overview` icon colors, that's why I removed it.

File: plugins/main/public/utils/applications.ts (hunk 8 above)
Comment (yenienserrano) [reply plugins/main/public/utils/applications.ts]: done [b1a4838](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6086/commits/b1a483869e88f2ab865dfddf90829d9628153202)

File: plugins/main/public/utils/applications.ts (hunk 7 above)
Comment (yenienserrano) [reply plugins/main/public/utils/applications.ts]: done [b1a4838](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6086/commits/b1a483869e88f2ab865dfddf90829d9628153202)

File: plugins/main/public/utils/applications.ts (hunk 6 above)
Comment (yenienserrano) [reply plugins/main/public/utils/applications.ts]: done [b1a4838](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6086/commits/b1a483869e88f2ab865dfddf90829d9628153202)

File: plugins/main/public/utils/applications.ts (hunk 3 above)
Comment (yenienserrano) [reply plugins/main/public/utils/applications.ts]: done [b1a4838](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6086/commits/b1a483869e88f2ab865dfddf90829d9628153202)

**Comment by asteriscos** [comment]
### Test

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome |
| --- |  --- |
| See the menu titles without camel case and the sub sections in camel case, in the breadcrumb it should look the same.| :black_circle: |
| The Home > Overview icon styles should be the same as those in the menu (test it in dark and light mode) | :black_circle: |
| Breadcrumb redirect well to the agent | :black_circle: |
| In the agent view: The pinned applications should redirect to the respective application without reloading the page and if you hover it should show the url to which it redirects. | :black_circle: |
| In the agent view: When the menu is opened with the `More` button the options should redirect to the respective application without reloading the page and if hovered the url to which it redirects should appear. | :black_circle: |
| In the agent view: The Open MITRE ATT&CK button should redirect to the MITRE ATT&CK application without reloading the page and if hovered the url to which it redirects should appear. | :black_circle: |
| In the agent view: On the `FIM` card the icon should redirect to `FIM` without reloading the page and if hovered the url should appear. | :black_circle: |
| In the agent view: On the `SCA: last scan` card the icon should redirect to `SCA` without reloading the page and if the cursor is hovered over the url it redirects to should appear. | :black_circle: |
| In the agent view: On the `SCA: last scan` card the title should redirect to `SCA` without reloading the page and hovering over it should display the url it redirects to. | :black_circle: |
| In the agent view: On the `SCA: last scan` card The caption should redirect to `SCA` without reloading the page and if hovered over should display the url to which it redirects. | :black_circle: |
| In the overview: Without agents the `Add agent` button should redirect to the `Endpoint Summary` application without making a page reload and if hovered the url to which it redirects should appear. | :black_circle: |
| In the overview:  The cards should redirect to their respective application without reloading the page and if you hover on the title you should see the url you are redirected to. | :black_circle: |
| In the healthcheck: The `Go to settings` button should redirect to the `Server APIs` application without reloading the page and if you hover it should show the url to which it redirects. | :black_circle: |
| Applications with sample data: The button that takes you to sample data from the applications should redirect to the `Sample Data` application without making a page reload and if you hover it should appear the url to which it redirects. | :black_circle: |
| In Endpoint Summary: The action buttons of the table should redirect to the `IT Hygiene` application without reloading the page and if hovered the url to which it redirects should appear. | :black_circle: |
| In the configuration of an agent: The group buttons should redirect to the `Endpoint Groups` application without reloading the page and if hovered the url should be displayed. | :black_circle: |
| In statistics: The `Settings` button should redirect to the `App Settings` application without a page reload and if hovered the url to which it redirects should appear. | :black_circle: |


**Details**
<details>
<summary>:black_circle: See the menu titles without camel case and the sub sections in camel case, in the breadcrumb it should look the same.
</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: The Home > Overview icon styles should be the same as those in the menu (test it in dark and light mode)</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: Breadcrumb redirect well to the agent</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: The pinned applications should redirect to the respective application without reloading the page and if you hover it should show the url to which it redirects.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: When the menu is opened with the `More` button the options should redirect to the respective application without reloading the page and if hovered the url to which it redirects should appear.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: The Open MITRE ATT&CK button should redirect to the MITRE ATT&CK application without reloading the page and if hovered the url to which it redirects should appear.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: On the `FIM` card the icon should redirect to `FIM` without reloading the page and if hovered the url should appear.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: On the `SCA: last scan` card the icon should redirect to `SCA` without reloading the page and if the cursor is hovered over the url it redirects to should appear.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: On the `SCA: last scan` card the title should redirect to `SCA` without reloading the page and hovering over it should display the url it redirects to.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the agent view: On the `SCA: last scan` card The caption should redirect to `SCA` without reloading the page and if hovered over should display the url to which it redirects.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the overview: Without agents the `Add agent` button should redirect to the `Endpoint Summary` application without making a page reload and if hovered the url to which it redirects should appear.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the overview:  The cards should redirect to their respective application without reloading the page and if you hover on the title you should see the url you are redirected to.
</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: In the healthcheck: The `Go to settings` button should redirect to the `Server APIs` application without reloading the page and if you hover it should show the url to which it redirects.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: - Applications with sample data: The button that takes you to sample data from the applications should redirect to the `Sample Data` application without making a page reload and if you hover it should appear the url to which it redirects.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: - In Endpoint Summary: The action buttons of the table should redirect to the `IT Hygiene` application without reloading the page and if hovered the url to which it redirects should appear.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: - In the configuration of an agent: The group buttons should redirect to the `Endpoint Groups` application without reloading the page and if hovered the url should be displayed.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: - In statistics: The `Settings` button should redirect to the `App Settings` application without a page reload and if hovered the url to which it redirects should appear.</summary>

Chrome - :black_circle:

</details>

**Comment by chantal-kelm** [comment]
In progress


| Test | Chrome |
| --- |  --- |
| See the menu titles without camel case and the sub sections in camel case, in the breadcrumb it should look the same.| :black_circle: |
| The Home > Overview icon styles should be the same as those in the menu (test it in dark and light mode) | :black_circle: |
| Breadcrumb redirect well to the agent | :black_circle: |
| In the agent view: The pinned applications should redirect to the respective application without reloading the page and if you hover it should show the url to which it redirects. | :black_circle: |
| In the agent view: When the menu is opened with the `More` button the options should redirect to the respective application without reloading the page and if hovered the url to which it redirects should appear. | :black_circle: |
| In the agent view: The Open MITRE ATT&CK button should redirect to the MITRE ATT&CK application without reloading the page and if hovered the url to which it redirects should appear. | :black_circle: |
| In the agent view: On the `FIM` card the icon should redirect to `FIM` without reloading the page and if hovered the url should appear. | :black_circle: |
| In the agent view: On the `SCA: last scan` card the icon should redirect to `SCA` without reloading the page and if the cursor is hovered over the url it redirects to should appear. | :black_circle: |
| In the agent view: On the `SCA: last scan` card the title should redirect to `SCA` without reloading the page and hovering over it should display the url it redirects to. | :black_circle: |
| In the agent view: On the `SCA: last scan` card The caption should redirect to `SCA` without reloading the page and if hovered over should display the url to which it redirects. | :black_circle: |
| In the overview: Without agents the `Add agent` button should redirect to the `Endpoint Summary` application without making a page reload and if hovered the url to which it redirects should appear. | :black_circle: |
| In the overview:  The cards should redirect to their respective application without reloading the page and if you hover on the title you should see the url you are redirected to. | :black_circle: |
| In the healthcheck: The `Go to settings` button should redirect to the `Server APIs` application without reloading the page and if you hover it should show the url to which it redirects. | :black_circle: |
| Applications with sample data: The button that takes you to sample data from the applications should redirect to the `Sample Data` application without making a page reload and if you hover it should appear the url to which it redirects. | :black_circle: |
| In Endpoint Summary: The action buttons of the table should redirect to the `IT Hygiene` application without reloading the page and if hovered the url to which it redirects should appear. | :black_circle: |
| In the configuration of an agent: The group buttons should redirect to the `Endpoint Groups` application without reloading the page and if hovered the url should be displayed. | :black_circle: |
| In statistics: The `Settings` button should redirect to the `App Settings` application without a page reload and if hovered the url to which it redirects should appear. | :black_circle: |


**Details**
<details>
<summary>:black_circle: See the menu titles without camel case and the sub sections in camel case, in the breadcrumb it should look the same.
</summary>

Chrome - :green_circle:

image

</details>

<details>
<summary>:black_circle: The Home > Overview icon styles should be the same as those in the menu (test it in dark and light mode)</summary>

Chrome - :green_circle:
image
image
image
</details>

<details>
<summary>:black_circle: Breadcrumb redirect well to the agent</summary>

Chrome - :green_circle:

https://github.com/wazuh/wazuh-dashboard-plugins/assets/99441266/31f8b70b-0899-48ca-bbd4-8d5999900b38


</details>

<details>
<summary>:black_circle: In the agent view: The pinned applications should redirect to the respective application without reloading the page and if you hover it should show the url to which it redirects.</summary>

Chrome - :green_circle:

image
image

</details>

<details>
<summary>:green_circle: In the agent view: When the menu is opened with the `More` button the options should redirect to the respective application without reloading the page and if hovered the url to which it redirects should appear.</summary>

Chrome - :green_circle:

image

</details>

<details>
<summary>:green_circle: In the agent view: The Open MITRE ATT&CK button should redirect to the MITRE ATT&CK application without reloading the page and if hovered the url to which it redirects should appear.</summary>

Chrome - :green_circle:

image


</details>

<details>
<summary>:green_circle: In the agent view: On the `FIM` card the icon should redirect to `FIM` without reloading the page and if hovered the url should appear.</summary>

Chrome - :green_circle:
image
image

</details>

<details>
<summary>:green_circle: In the agent view: On the `SCA: last scan` card the icon should redirect to `SCA` without reloading the page and if the cursor is hovered over the url it redirects to should appear.</summary>

Chrome - :green_circle:
image
image

</details>

<details>
<summary>:green_circle: In the agent view: On the `SCA: last scan` card the title should redirect to `SCA` without reloading the page and hovering over it should display the url it redirects to.</summary>

Chrome - :green_circle:

image
image

</details>

<details>
<summary>:green_circle: In the agent view: On the `SCA: last scan` card The caption should redirect to `SCA` without reloading the page and if hovered over should display the url to which it redirects.</summary>

Chrome - :green_circle:
image
image

</details>

<details>
<summary>:green_circle: In the overview: Without agents the `Add agent` button should redirect to the `Endpoint Summary` application without making a page reload and if hovered the url to which it redirects should appear.</summary>

Chrome - :green_circle:

</details>

<details>
<summary>:black_circle: In the overview:  The cards should redirect to their respective application without reloading the page and if you hover on the title you should see the url you are redirected to.
</summary>

Chrome - :black_circle:

https://github.com/wazuh/wazuh-dashboard-plugins/assets/99441266/056c2a10-68d3-40f1-a102-2a4ef39a65f9




</details>

<details>
<summary>:black_circle: In the healthcheck: The `Go to settings` button should redirect to the `Server APIs` application without reloading the page and if you hover it should show the url to which it redirects.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: - Applications with sample data: The button that takes you to sample data from the applications should redirect to the `Sample Data` application without making a page reload and if you hover it should appear the url to which it redirects.</summary>

Chrome - :black_circle:
image

</details>

<details>
<summary>:black_circle: - In Endpoint Summary: The action buttons of the table should redirect to the `IT Hygiene` application without reloading the page and if hovered the url to which it redirects should appear.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: - In the configuration of an agent: The group buttons should redirect to the `Endpoint Groups` application without reloading the page and if hovered the url should be displayed.</summary>

Chrome - :black_circle:

</details>

<details>
<summary>:black_circle: - In statistics: The `Settings` button should redirect to the `App Settings` application without a page reload and if hovered the url to which it redirects should appear.</summary>

Chrome - :black_circle:

</details>

**Review by asteriscos — CHANGES_REQUESTED** [review]
CR: :green_circle: 
Test: :yellow_circle:  The breadcrumb flickers and changes its state several times on each navigation. This should be fixed.

**Review by chantal-kelm — COMMENTED** [review]
Code review: 🟢 

Test: 🟡 The breadcrumb flickers and changes its state several times on each navigation. This should be fixed.

**Comment by Desvelao** [comment]
LGTM!

**Comment by Desvelao** [comment]
> CR: 🟢 Test: 🟡 The breadcrumb flickers and changes its state several times on each navigation. This should be fixed.

The flick is caused due to the navigation between applications. This is happening with built-in applications too.

**Review by asteriscos — APPROVED** [review]
The flicker issue is not because of this pull request. So I will approve the PR. :heavy_check_mark:
